leafs/TalkingLeaf.js

```diff
@@ -10,7 +10,7 @@ class TalkingLeaf extends BSClientLeaf {
      * @param {AudioContext} [audioContext=new AudioContext()]
```

Comment:
I've not updated the JSDoc here on purpose: This is what's happening in `ClientMouth` if `audioContext` is undefined here.